spec/system/jobseekers/school_search_mobile_display_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+RSpec.describe "Jobseeker job search filter mobile display" do
+  let(:request) { visit organisations_path(params) }
+  let(:params) { {} }
+  let(:base_selector) { ".panel-component.js-action" }
+
+  before do
+    Capybara.current_driver = :rack_test
```

Comment:
I'm not sure that this is needed? The default should be rack_test, and :js tag should convert it to selenium...?